src/demo/demo.js

```diff
@@ -6,16 +6,18 @@ import { BrowserRouter as Router, Route, Link } from 'react-router-dom'
 /*
 The SDK can be consumed either via npm or via global window.
 Via npm there are also two ways, via commonjs require or via ES import.
-In this case we will use the import via the global `window`.
+In this case we will use the "es" import style.
 
 Alternative import styles:
 "commonjs" import style
 const Onfido = require('../index.js')
 "es" import style
 import * as Onfido from '../index.js'
+global `window` import
+const Onfido = window.Onfido
 */
 
-const Onfido = window.Onfido
+import * as Onfido from '../index.js'
```

Comment:
I don't think this import method works as expected with this setup. 
The SDK doesn't get initialised properly so modal mode is broken (nothing gets rendered in modal view) and Country Selection shows up as the next step when Passport document type is selected.

Reply:
I cannot reproduce the issues you're mentioning. Did you run that locally?

Reply:
Yeah, ran `npm install` then `npm run dev`

BTW when I run `npm install` there's also a console warning on Terminal
```
console.warn src/components/utils/index.js:10
    The css @value: undefined unit is undefined but it should be ms
```